src/main/java/pages/YandexLoginPage.java

```diff
@@ -1,11 +1,52 @@
 package pages;
 
+import configuration.DriverManager;
 import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
 
 public class YandexLoginPage {
     public final static By ENTER_BUTTON = By.cssSelector(".Button2_view_default.Button2_size_m");
     public final static By LOGIN_INPUT = By.id("passp-field-login");
     public final static By LOGIN_BUTTON = By.id("passp:sign-in");
     public final static By PASSWORD_INPUT = By.name("passwd");
     public final static By ACCOUNT_NAME = By.xpath("//a[contains(@class, 'user-account_left' )]/span[@class='user-account__name']");
-}
+    public final static By ACCOUNT_ICON = By.xpath("//a[contains(@class, 'user-account_left' )]/span[@class='user-account__name']/..");
+    public final static By LOGOUT_BUTTON = By.xpath("//a[@aria-label='Log out']");
```

Comment:
do you really have this elements in the LoginPage? It's better to separate classes according to pages. Ex.: `LoginPage`, `UserAccountPage`, `MailPage `etc